### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by pvuorela**
Bunch of compiler warnings here, inconsistent style and even a wayland scanner warning for failed validation.

### CODE REVIEW

File: src/screentovnc.cpp [hunk 1]
```diff
@@ -167,7 +191,7 @@ ScreenToVnc::ScreenToVnc(QObject *parent, bool smoothScaling, float scalingFacto
     char *argv[0];
     m_server = rfbGetScreen(0,argv, m_screen_width, m_screen_height, 8, 3, m_screen->depth() / 8);
```
Comment (spiiroin) [inline src/screentovnc.cpp:192]: old code nit: this bogus argv thing here feels so wrong... if rfbGetScreen(0, nullptr, ...) is not okay, then char *argv[1] = { nullptr }; would be more as expected...

File: src/screentovnc.cpp [hunk 2]
```diff
@@ -179,21 +203,21 @@ ScreenToVnc::ScreenToVnc(QObject *parent, bool smoothScaling, float scalingFacto
...
         uinputCreateKeyboardDevice();
         m_server->kbdAddEvent = keyboardHandler;
     }
 
     // check if launched by systemd with a ready socket (LISTEN_FDS env var)
     int sd_fds = sd_listen_fds(1);
-    if (sd_fds){
-        for (int i = SD_LISTEN_FDS_START; i <= (SD_LISTEN_FDS_START + sd_fds - 1); i++){
+    if (sd_fds) {
+        for (int i = SD_LISTEN_FDS_START; i <= (SD_LISTEN_FDS_START + sd_fds - 1); i++) {
```
Comment (spiiroin) [inline src/screentovnc.cpp:218]: eh? nit: could be: i < SD_LISTEN_FDS_START + sd_fds

File: src/screentovnc.cpp [hunk 3]
```diff
@@ -972,64 +995,64 @@ void ScreenToVnc::mouseHandler(int buttonMask, int x, int y, rfbClientPtr cl)
...
+            if (hasAbsMtWidthMajor) {
                 event_mt_width_major.type = EV_ABS;
                 event_mt_width_major.code = ABS_MT_WIDTH_MAJOR;
                 event_mt_width_major.value = 0;
             }
 
-            if (hasBntTouch){
-                if(write(eventDev, &event_btn_touch, sizeof(event_btn_touch)) < sizeof(event_btn_touch)) {
+            if (hasBntTouch) {
+                if (write(eventDev, &event_btn_touch, sizeof(event_btn_touch)) < sizeof(event_btn_touch)) {
```
Comment (spiiroin) [inline src/screentovnc.cpp:1023]: Are there no signed/unsigned warnings from these? afaik ssize_t from write() vs size_t from sizeof() -> signed value is promoted to unsigned for comparing -> -1 return values becomes something that is unlikely to be caught as an error...
Or am I missing something? Like is write() here something else than ::write()?

**Review by spiiroin — APPROVED** [review]
Had some already-in-old-code / cosmetic nits but did not spot any actual problems.

File: src/screentovnc.cpp (hunk 3 above)
Comment (pvuorela) [reply src/screentovnc.cpp:1023]: There are but I just thought the fixes currently would be sufficient for this round :)

File: src/screentovnc.cpp (hunk 1 above)
Comment (pvuorela) [reply src/screentovnc.cpp:192]: Though it's given explicit argc of 0. Dunno, maybe I'll avoid touching now.

File: src/screentovnc.cpp (hunk 2 above)
Comment (pvuorela) [reply src/screentovnc.cpp:218]: True.

**Pull request merged**